lib/stripe/checkout/session.ex

```diff
@@ -0,0 +1,104 @@
+defmodule Stripe.Session do
+  @moduledoc """
+  Work with Stripe Checkout Session objects.
+
+  You can:
+
+  - Create a new session
+
+  Stripe API reference: https://stripe.com/docs/api/checkout/sessions
+  """
+
+  use Stripe.Entity
+  import Stripe.Request
+
+  @type line_item :: %{
```

Comment:
We have a `Stripe.LineItem` object.  Has everything except `images`.  Don't see it [here](https://stripe.com/docs/api/invoices/line_item) though 🤔

Reply:
I think that might be a new feature for the new Stripe Checkout: https://stripe.com/docs/api/checkout/sessions/create#create_checkout_session-line_items

My personal thinking is that the `Stripe.LineItem` type has a lot of fields on it that won't be accepted by this endpoint, so it would be better to have a second local `line_item` type that is specific to this use case. If you prefer though, I'm happy to add an `images` field to `Stripe.LineItem` and use that.

Reply:
na we can leave it as is 👍